src/views/TechnicalAssistance/RequestForm/DocumentsTable.tsx

```diff
@@ -129,6 +141,30 @@ function DocumentsTable({
   ]);
 
   const columns = useMemo<Column<TrbRequestDocuments>[]>(() => {
+    const getUrlForDocument = (documentId: string) => {
```

Comment:
[Slack thread](https://oddball.slack.com/archives/C037ZNEQ7DH/p1688592470124689) for discussing with the frontend team if this is the right approach, as well as how to handle errors.

Reply:
From conversation with @adamodd, using the Promise-based approach seems to be decent. https://github.com/CMSgov/easi-app/pull/2095/commits/7cd4ff71f5428de7407b19718a2e228e8f6e96c2 adds error handling by showing an alert with `showMessage()`; this works in the admin view, but doesn't display anything in the requester view.

Reply:
Going with this approach.